Sources/MongoSwift/ClientSession.swift

```diff
@@ -216,3 +173,57 @@ public final class SyncClientSession {
         }
     }
 }
+
+/**
+ * A MongoDB client session for use with synchronous clients, databases, and sessions.
+ * This class represents a logical session used for ordering sequential operations.
+ *
+ * To create a client session, use `startSession` or `withSession` on a `SyncMongoClient`.
+ *
+ * If `causalConsistency` is not set to `false` when starting a session, read and write operations that use the session
+ * will be provided causal consistency guarantees depending on the read and write concerns used. Using "majority"
+ * read and write preferences will provide the full set of guarantees. See
+ * https://docs.mongodb.com/manual/core/read-isolation-consistency-recency/#sessions for more details.
+ *
+ * e.g.
+ *   ```
```

Comment:
I wonder if jazzy supports the `swift` word here for syntax highlighting, or if it does it automatically.